docs/guide/migrating-to-version-9.md

```diff
@@ -110,6 +122,31 @@ No changes.
 
 ### KInput
 
+#### Structure
+
+* `form-control` class is no longer present
+* `over-char-limit` class is no longer present
+* `has-error` class is no longer present
+* `help` class has been changed to `help-text`
+
+#### Constants, Types & Interfaces
+
+* `SizeArray` and `IconPositionArray` constants are no longer exported
+* `Size` and `IconPosition` types are no longer exported
```

Comment:
question: are they no longer exported, or have been removed?

Reply:
Changed this to `have been removed`